### ISSUE TITLE
Option to block all verified Youtube accounts?

### ISSUE THREAD

**Issue by TomArrow** [miscavage/De-Mainstream-YouTube-Extension#33]
A bit of an odd request I guess, but can you add an option to filter all Youtube videos from an account that is verified? They all have that little "checkmark" next to the channel name. This could be useful in case of major news events, like hurricanes and such, to find original source videos from normal people instead of the hundreds of reuploads by greedy big channels. Obviously this would also catch a few "good" big channels, but it could be deactivated by default and only activated for special cases like that.

**Comment by DonaldTsang** [comment]
This sounded useful but then we would need a whitelist

**Comment by TomArrow** [comment]
> 
> 
> This sounded useful but then we would need a whitelist

Not necessarily, it could be off by default and only activate for specific situations, like when you are searching for really small channels.

**Comment by DonaldTsang** [comment]
Natural vs Astroturf?

**Comment by TomArrow** [comment]
Yeah I guess you could say it like that.
